presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java

```diff
@@ -148,29 +154,102 @@ private static long epochDaysInZone(TimeZoneKey timeZoneKey, long instant)
     public void testLocalTime()
             throws Exception
     {
-        long millis = new LocalTime(session.getStartTime(), DateTimeZone.UTC).getMillisOfDay();
-        functionAssertions.assertFunction("LOCALTIME", TimeType.TIME, toTime(millis));
+        Session localSession = testSessionBuilder()
+                .setSystemProperty("legacy_timestamp", "false")
+                .setTimeZoneKey(TIME_ZONE_KEY)
```

Comment:
You shouldn't need to set `legacy_timestamp` and TZ in tests method separately. Just create session build from `this.session`.

(here & elsewhere)